src/orchestration/queue_test.go

```diff
@@ -14,6 +14,46 @@ import (
 	"time"
 )
 
+func Test_NewQueueHandler_FailsToCreateClientFromConnectionString_ReturnsError(t *testing.T) {
+	os.Setenv("AZURE_STORAGE_CONNECTION_STRING", "")
+	defer os.Unsetenv("AZURE_STORAGE_CONNECTION_STRING")
+
+	defaultLogger := slog.Default()
+	defer slog.SetDefault(defaultLogger)
```

Comment:
Why are we deferring the slogger here? Does that actually do anything?

Reply:
I believe that's how we can assert whether the slogging matches the expected output once the function finishes. @halprin, right?

Reply:
Correct.  We save the original, default logger, and then we defer setting the default back to the original because we're about to change the default below this comment.